src/lib/__tests__/Preview-test.js

```diff
@@ -1715,27 +1713,16 @@ describe('lib/Preview', () => {
 
         test('should show the loading download button if file can be downloaded', () => {
             stubs.canDownload.mockReturnValue(true);
-            preview.loadViewer();
+            preview.loadViewer({});
```

Comment:
Is the empty object argument needed?

Reply:
No, but that's how it is in master. I'm not making any changes in this area, so I reverted this change as well.